src/commands/addWalletIdentityCommand.ts

```diff
@@ -207,19 +207,22 @@ export async function addWalletIdentity(walletItem: WalletTreeItem | IFabricWall
             // Get a list of CAs
             const caKeys: string[] = Object.keys(connectionProfile.certificateAuthorities);
             let caUrl: string;
+            let caName: string;
 
             if (caKeys.length > 1) {
                 const showMoreCAs: string = await UserInputUtil.showQuickPickCA(caKeys);
                 if (!showMoreCAs) {
                     return;
                 } else {
                     caUrl = connectionProfile.certificateAuthorities[showMoreCAs].url;
+                    caName = connectionProfile.certificateAuthorities[showMoreCAs].caName;
                 }
             } else {
                 caUrl = connectionProfile.certificateAuthorities[caKeys[0]].url;
+                caName = connectionProfile.certificateAuthorities[caKeys[0]].caName;
             }
 
-            const enrollment: { certificate: string, privateKey: string } = await certificateAuthority.enroll(caUrl, enrollmentID, enrollmentSecret);
+            const enrollment: { certificate: string, privateKey: string } = await certificateAuthority.enroll(caUrl, enrollmentID, enrollmentSecret, caName);
```

Comment:
Would it be possible to update the tests [here](https://github.com/IBM-Blockchain/blockchain-vscode-extension/blob/master/test/commands/addWalletIdentityCommand.test.ts) to check that `enroll` is being called with the `caName` as well?

Reply:
Sure. I will try to add a test case and then update this pr.